app/src/mainwindow2.cpp

```diff
@@ -196,6 +200,7 @@ void MainWindow2::createDockWidgets()
     addDockWidget(Qt::RightDockWidgetArea, mColorBox);
     addDockWidget(Qt::RightDockWidgetArea, mColorInspector);
     addDockWidget(Qt::RightDockWidgetArea, mColorPalette);
+    addDockWidget(Qt::RightDockWidgetArea, mXsheet);
```

Comment:
I agree that the xsheet should be hidden by default. Pencil2D should keep its clean and easily accessible interface.
I also agree that the xsheet should be updated more efficient. I know there is a wish that the frames should be visible (in miniature) in the xsheet, so it will be corrected. That said, I think 60 fps or more i only seen in games, where it is common to have 50-100 fps, but that depends on cpu power. Hand drawn animation is max 24 or 25 fps in Europe and 30 fps in the States. Correct me if I'm wrong.

Reply:
I think I've solved the problems mentioned. Rewrites of xsheet is now limited to situations where it is necessary.

Reply:
@scribblemaniac  I have made changes, so I use the layer index in stead of layer name as identifier. This means that identical layer names are supported now.